.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
```

Comment:
RuboCop TODOs were regenerated since refactoring the Workflow model to make the class smaller is out of scope for this PR.